bpo-45735: Promise the long-time truth that `args=list` works (#30982)
python/cpython: Doc/library/multiprocessing.rst

   to ``True`` or ``False``.  If ``None`` (the default), this flag will be
   inherited from the creating process.

   By default, no arguments are passed to *target*. The *args* argument,

>    which defaults to ``()``, can be used to specify a list or  tuple of  the arguments

Reviewer: nitpick: two spaces in here?
![image](https://user-images.githubusercontent.com/9103416/153526933-6ed56aa3-935a-4810-8168-3300e9be1ef0.png)

Author: Thanks. I've removed redundant spaces.

>    By default, no arguments are passed to *target*, the *args* is set to ``()``.
>    Besides, similar to :class:`threading.Thread`, it is feasible to use a list or
>    tuple as the *args* to pass arguments.

Reviewer 2: ```suggestion
   By default, no arguments are passed to *target*. The *args* argument,
   which defaults to ``()``, can be used to specify a list or  tuple of  the arguments
   to pass to *target*.
```
Cleaner Englishl.

      the target argument, if any, with sequential and keyword arguments taken
      from the *args* and *kwargs* arguments, respectively.

>       Using list or tuple as the *args* argument which passed to the :class:`Process`
>       could achieve the same effect.

Reviewer 2: ```suggestion
      Using a list or tuple as the *args* argument passed to :class:`Process`
      achieves the same effect.
```
Cleaner English here.
